_pages/home/updating-to-version-5/updating-to-version-5.md

```diff
@@ -70,10 +79,94 @@ With the release of Spartacus 5.0, the hosting service of SAP Commerce Cloud can
 
    For more information, see [Adding Applications for JavaScript Storefronts](https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/b2f400d4c0414461a4bb7e115dccd779/63577f67a67347bf9f4765a5385ead33.html).
 
+## Removal of ng-bootstrap
+
+ng-bootstrap has been removed from Spartacus 5.0 and is no longer a dependency. You can continue to use ng-bootstrap in your Spartacus application, but this may require upgrading additional third party dependencies. For more information, see the list of [ng-bootstrap dependencies](https://www.npmjs.com/package/@ng-bootstrap/ng-bootstrap).
```

Comment:
Nitpick :) 

Maybe let's consider to name change to NgBootstrap or NG Bootstrap as it is name on https://www.npmjs.com/package/@ng-bootstrap/ng-bootstrap

ng-bootstrap refers more to package name.

What do you think?

Reply:
Good catch! I'll definitely change this name to NG Bootstrap 👌